README.md

```diff
@@ -11,10 +11,11 @@ Note: Use at your own risk, as this is an early version that is expected to unde
 
 To compile this project, you will need to follow these steps:
 
-1. Copy everything in the ``Kerbal Space Program 2\KSP2_x64_Data\Managed`` folder into the ``external_dlls/`` folder.
+1. Copy everything in the ``Kerbal Space Program 2\KSP2_x64_Data\Managed`` folder into the ``external_dlls`` folder.
 2. Run one of the build scripts and copy the contents from the build to the KSP2 root directory.
-3. Launch KSP2 and wait until the title screen appears. You should see a mods folder under the `KSP2_X64_data` folder.
-4. Drag any mods that follow the structure below into that mods folder.
+3. Create a mods folder at ``Kerbal Space Program 2\SpaceWarp\Mods``.
```

Comment:
SpaceWarp creates this folder, does it not?

Reply:
It *should* sometimes it doesn't, will provide a mods folder into the build folder next version anyways